src/main/java/org/example/Board.java

```diff
@@ -1,76 +1,98 @@
 package org.example;
+import lombok.Getter;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 
+@Getter
 public class Board {
-    private List<Integer> intList ;
-    private static final String RED = "\u001B[31m";
-    private static final String GREEN = "\u001B[32m";
-    private static final String WHITE = "\u001B[0m";
-    private char[][] board ;
+  private List<Integer> emptyPositions;
+  private static final char X_SYMBOL = 'X';
+  private static final char O_SYMBOL = 'O';
+  private static final String SEPARATOR = " | ";
+  private static final String RED = "\u001B[31m";
+  private static final String GREEN = "\u001B[32m";
+  private static final String WHITE = "\u001B[0m";
+  private char[][] board;
```

Comment:
Size of the board is not defined anywhere. Try to use BOARD_SIZE variable and ask user to give input for boardsize so that you can use it for nXn game too.